content/docs/composition-vs-inheritance.md

```diff
@@ -1,22 +1,22 @@
 ---
 id: composition-vs-inheritance
-title: Composition vs Inheritance
+title: Կոմպոզիցիան ընդդեմ Ժառանգականության
 permalink: docs/composition-vs-inheritance.html
 redirect_from:
   - "docs/multiple-components.html"
 prev: lifting-state-up.html
 next: thinking-in-react.html
 ---
 
-React has a powerful composition model, and we recommend using composition instead of inheritance to reuse code between components.
+React-ը ունի կոմպոզիցիայի հզոր մոդել, և մենք խորհուրդ ենք տալիս օգտագործել կոմպոզիցիա ժառանգականության փոխարեն\` կոմպոնենտների միջև կոդ վերա-օգտագործելու համար։
 
-In this section, we will consider a few problems where developers new to React often reach for inheritance, and show how we can solve them with composition.
+Այս գլխում, մենք կդիտարկենք մի քանի խնդիրներ, որտեղ ծրագրավորողները, որոնք նորեկ են React-ում, հաճախ հասնում են ժառանգականության միջոցով, և ցույց կտանք\` ինչպես լուծել այդ խնդիրները կոմպոզիցիայի միջոցով։
```

Comment:
"reach for inheritance"-ը " չեմ հասկանում, բառացի "հասնում են ժառանգականության համար" ա, ոչ թե "միջացով"։ Բայց իմաստային քո գրածը ճիշտ ա թվում, եթե վստահ ես, ուրեմն ասելիք չունեմ

Reply:
"որտեղ"֊ը կարողա՞ "որոնց"։ Խնդիրներ, որոնց հասնում են

Reply:
հանգելն էլ ավելի կարողա սազեր հասնելու փոխարեն